MainPage.xaml

```diff
@@ -35,12 +34,13 @@
                     >
             <Button Content="LoadThis" Click="LoadThisClicked" />
             <ContentPresenter ContentTemplate="{StaticResource RemoteSystemModelDisplay}" 
-                              Content="{Binding ThisRemoteSystem}"
+                              Content="{x:Bind _model.ThisRemoteSystem, Mode=OneWay}"
+                              Visibility="{x:Bind CollapsedIfNull(_model.ThisRemoteSystem), Mode=OneWay}"
```

Comment:
Oh fancy... how does one actually discover this? I must of missed it in the x:Bind docs?